Python/Dictionary_Comprehension/dictionary_comprehension_video.md

```diff
@@ -0,0 +1,2 @@
+## Dictionary Comprehension Video
```

Comment:
@rex2828 Great work on the video!! 
- Just mention the topics covered in your md file
- Add the relative link of your file to the readme of the folder

Reply:
@apeksha235 Dictionary Comprehension folder is already added in the Readme, do you want me to add the seperate link for my md file or what?